vispy/geometry/parametric.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Nicolas P. Rougier
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+from .normals import normals
```

Comment:
Did you delete this file? It looks like you did (since it was in the last review but not this one), but the ref is still here...?